examples/react-app/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": true,
+  "printWidth": 80,
+  "plugins": ["prettier-plugin-tailwindcss"]
+}
```

Comment:
On the root project we already have biome.js configure, can we remove the prettier configs?